src/views/Salon/salon.styled.ts

```diff
@@ -0,0 +1,66 @@
+import { styled } from '@mui/system';
+import { Box, IconButton, Typography } from '@mui/material';
+import theme from 'theme';
+import { LinkProps } from 'react-router-dom';
+
+export const CustomWrapper = styled(Box)(() => ({
```

Comment:
Why so many pixel units in those styles?

Reply:
I've tried to remove all unnecessary styles. Others that remain are required to ensure the correct display of our component.